### DISCUSSION TITLE
Speed Comparison

### DISCUSSION THREAD

**Discussion by GaudiestTooth17** [Axect/Peroxide#42] [Q&A]
How does the speed of the pure rust implementation of Peroxide compare to NumPy? Or Julia? I'm just trying to get a feel for how much of a speed boost I could get from using this library. :)

**Comment by Axect** [comment]
There is a simple benchmark - [Scientific_Bench](https://github.com/Axect/Scientific_Bench).
But it contains only matrix multiplication & basic summation. I'll work on more detailed benchmark. (But I'm too busy right now, so I don't have time to work.  :sob: )

Of course, you can do benchmark for any numerical stuff yourself. It's good to see [Peroxide_Gallery](https://github.com/Axect/Peroxide_Gallery).
